packages/edit-site/src/components/global-styles/font-library-modal/upload-fonts.js

```diff
@@ -1,19 +1,214 @@
 /**
```

Comment:
This file has been renamed from local-fonts.js to upload-fonts.js, as local-fonts.js was just a container with a spacer that imported upload-fonts.js. The spacer is no longer needed, as it looks like it was only there to provide spacing between the notice and the upload component, and this is now handled in `TabPanelLayout`.

Reply:
It took me a minute to understand that all of the changes were from the removed 'local-fonts.js`.  Thank you for explaining what happened. :)

Reply:
A few pending PRs have now conflicts because of this file renaming. It took me a while to research and understand what happened. Thank you for explaining.it. Mentioning it in the PR description would be even more appreciated, thanks,

Reply:
Updated the PR description to include this info as well, thanks!